addons/l10n_it_edi_sdicoop/models/account_invoice.py

```diff
@@ -22,3 +22,9 @@ def send_pec_mail(self):
         # With SdiCoop web-service, no need to send PEC mail.
         # Set the state to 'other' because the invoice should not be managed par l10n_it_edi.
         self.l10n_it_send_state = 'other'
+
+    @api.depends('restrict_mode_hash_table', 'state', 'l10n_it_edi_transaction')
```

Comment:
feedback from @jco-odoo
It's not necessary to depend on `'restrict_mode_hash_table', 'state'`, since the field in the inherited module already depends on them :smile:

Reply:
Oh, ok, je ne connais pas les détails internes.
Je sais que je ne suis pas le meilleure implémentateur mais j'aime beaucoup les bons résultats. 😄